optaplanner-solver/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-build-parent</artifactId>
+    <version>8.17.0-SNAPSHOT</version>
+    <relativePath>../build/optaplanner-build-parent/pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>optaplanner-solver</artifactId>
+  <description>Aggregate dependency to bring in optaplanner-core and all the score directors</description>
```

Comment:
name + url element missing.

We should bring consistency through all description elements in all our pom.xml (different PR). They end up on mvnrepository.com etc and influence SEO even. Making this more complex is the fact that code.quarkus.io uses the description element of optaplanner-quarkus's pom.xml.